README.md

```diff
@@ -1,51 +1,74 @@
 # regula-action
 
-[Regula] is a tool that evaluates Terraform infrastructure-as-code for potential security misconfigurations and compliance violations prior to deployment. This is a [GitHub Action] to run [Regula] against your repository.
+[Regula] is a tool that evaluates CloudFormation and Terraform infrastructure-as-code for potential AWS, Azure, and Google Cloud security misconfigurations and compliance violations prior to deployment. This is a [GitHub Action] to run [Regula] against your repository.
 
 ## Example
 
 Here's an example workflow file:
```

Comment:
```suggestion
Here's an example workflow file.  It checks three different IaC configurations: one Terraform directory and two CloudFormation templates:
```

Added this since people may wonder, why do they need three jobs?